src/main/java/com/intuit/graphql/orchestrator/utils/XtextToGraphQLJavaVisitor.java

```diff
@@ -74,9 +77,31 @@
   private final Map<String, GraphQLType> graphQLObjectTypes;
   public final Map<String, GraphQLDirective> directiveDefinitions;
 
+  private static final GraphQLScalarType FIELD_SET_SCALAR;
+
   static {
     STANDARD_SCALAR_TYPES = ScalarInfo.STANDARD_SCALARS.stream()
         .collect(Collectors.toMap(GraphQLScalarType::getName, Function.identity()));
+
+//    TODO REMOVE ONCE GRAPH-QL JAVA IS UPGRADED.....FIELD SET WILL ALREADY BE IN STANDARD SCALARS
```

Comment:
nit:  might be good to add a unit test that will fail once field set is already in standard scalars.